configuration.go

```diff
@@ -56,4 +56,10 @@ type ClientConfiguration struct {
 	// TokenCache is used to retrieve previously acquired tokens and saving new ones
 	// If not set we do not use a cache to store the new acquired tokens
 	TokenCache authenticator.TokenCache
+	// TokenExpiration defines lifetime of generated tokens
+	// Has no effect for tokens provided via the Token field
+	TokenExpiration string
+	// Whitelisted is used to indicate only whitelisted IP's may use generated tokens
+	// Has no effect for tokens provided via the Token field
+	Whitelisted bool
```

Comment:
I am not sure about the name `Whitelisted` in the client configuration as it is a bit unclear.

Reply:
I'm just copying terms already present in the code. Do you ave a suggestion for another name?

Reply:
At the moment I can't think of anything better than TokenWhitelisted or TokenWhitelistedIP.
Those are all pretty ugly, so I am resolving this.

Reply:
Adopted "TokenWhitelisted" to clarify it reflects the _token_ using the whitelist.